pkg/ccl/multitenantccl/tenantcostclient/tenant_side.go

```diff
@@ -150,6 +115,22 @@ const extendedReportingPeriodFactor = 4
 // estimated usage. This is intended to support usage spikes without blocking.
 const bufferTokens = 5000
 
+// defaultWriteBatchRate specifies the write batch rate that will be used to
+// estimate KV CPU when an actual measurement of the rate is not available.
+const defaultWriteBatchRate = 1000
+
+// tokensPerCPUSecond is the factor used to convert from estimated KV CPU
+// seconds to tokens in the distributed token bucket. This factor was chosen to
+// convert from seconds to milliseconds, as that measurement has a similar
+// magnitude as request units.
+const tokensPerCPUSecond = 1000
+
+// vCPUsPerNode is the number of vCPUs that each "node" in a tenant cluster is
+// assumed to have. This number is independent of the actual vCPUs per node in
+// the underlying host cluster, so that estimated CPU will not change when host
+// cluster hardware changes.
+const vCPUsPerNode = 8
```

Comment:
I think you can delete `vCPUsPerNode` now that the setting describes the number of nodes in the system.

Reply:
Done.